src/components/filters/FristFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import { CheckboxGroup, Checkbox } from '@navikt/ds-react';
+import { FristFilterOption } from '@/utils/hendelseFilteringUtils';
+import React from 'react';
+
+const texts = {
+  legend: 'Velg synlighet for frist',
```

Comment:
Hvor vises denne?

Reply:
Det er selve labelen for checkbox groupen. Akkurat denne har ikke kommet med i screenrecordingen over. Kom gjerne med forslag til endringer her også. Bare "Frist" følte jeg ble litt intetsigende, men kan godt gå for dette om vi tenker det er best.
Sånn ser det ut nå👇
<img width="301" alt="Screenshot 2024-01-05 at 12 15 38" src="https://github.com/navikt/syfooversikt/assets/11747383/d0ca60e1-95ae-4407-bbd6-12c3feeba472">

Reply:
Skjønner :+1: "Frist" er jo på linje med Virksomheter og Fødselsdato over da, synes kanskje ikke det var så åpenbart hva "Velg synlighet" betyr :thinking: